docs/runbook.md

```
@@ -44,198 +47,191 @@ from public.work_queue where status='pending';
 
 -- Stuck locks > 15 min
 select * from work_queue
 where status='pending' and locked_at < now() - interval '15 min';
-Remediate
+```
 
-Increase concurrency / instance count for the worker.
+**Remediate**
+- Increase concurrency / instance count for the worker.
+- Unlock stuck rows (use carefully):
 
```

markdownlint-cli2 (0.18.1)

Hard tabs
Column: 13

(MD010, no-hard-tabs)